### PR TITLE
Adding Icon dependency to chip component

### PR DESCRIPTION
**Pull request by CloudPower97**
# How Has This Been Tested?

Running `yarn test`

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

**Comment by alextrastero** [comment]
Same as before really, what are we trying to achieve here?

**Comment by alextrastero** [comment]
And it's adding a dependency to `Icon`

**Comment by CloudPower97** [comment]
@alextrastero Isn't adding the dependency to `Icon` a _good thing_? If in the future for some reason we change the rendering of icons the changes made will be also reflected here without the need to refactor this component.

**Comment by alextrastero** [comment]
Good point, still for a small component like this we don't need a `class`, what if we just replace the `<i className` with an `Icon`?

**Comment by CloudPower97** [comment]
@alextrastero right, going to do that ASAP.

**Comment by CloudPower97** [comment]
@alextrastero Sorry to being so late

File: src/Chip.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import PropTypes from 'prop-types';
 import cx from 'classnames';
+import Icon from './Icon';
 
 const Chip = ({ children, close, className, ...props }) => {
   return (
     <div className={cx('chip', className)} {...props}>
       {children}
-      {close ? <i className="close material-icons">close</i> : null}
+      {close && <Icon className="close material-icons">close</Icon>}
```
Comment (alextrastero) [inline src/Chip.js]: Hey @CloudPower97, just realised that `<Icon/>` component already includes these classes
```suggestion
      {close && <Icon>close</Icon>}
```
would suffice

Comment (CloudPower97) [reply src/Chip.js]: Whoops, you are right, It was my oversight

**Comment by CloudPower97** [comment]
@alextrastero ping

**Pull request merged**